cli/src/test/resources/ide-projects/settings/project/software/mvn/bin/readme

```diff
@@ -0,0 +1 @@
+this is the mvn bin folder
```

Comment:
This file can be removed. Only required if we want to read from it in a test. Please use `.gitkeep` instead if a folder is needed.